frontendVue3/src/components/Constraints.vue

```diff
@@ -173,11 +173,16 @@ export default {
             }
             this.closeAddEditDialog();
             this.commandManager.execute(command);
+            this.$emit('update-feature-model');
         },
 
         deleteConstraint(constraint) {
+            if(constraint.isHighlighted){ //reset highlight to free up color
```

Comment:
This works fine to workaround the Problem mentioned in #344, however has the drawback that undo/redo does not automatically highlight the deleted Constraint again.